tough/src/schema/mod.rs

```diff
@@ -171,12 +230,43 @@ pub struct Snapshot {
     pub _extra: HashMap<String, Value>,
 }
 
+/// Represents a metadata file in a `snapshot.json` file.
+/// TUF 4.4: METAFILES is an object whose format is the following:
+/// ```text
+///  { METAPATH : {
+///        "version" : VERSION,
+///        ("length" : LENGTH, |
+///         "hashes" : HASHES) }
+///    , ...
+///  }
+/// ```
+/// e.g.
+/// ```json
+///    "project1.json": {
+///     "version": 1,
+///     "hashes": {
+///      "sha256": "f592d072e1193688a686267e8e10d7257b4ebfcf28133350dae88362d82a0c8a"
+///     }
+///    },
+/// ```
 #[derive(Debug, Clone, Deserialize, Serialize, PartialEq)]
 pub struct SnapshotMeta {
+    /// TUF 4.4: LENGTH is the integer length in bytes of the metadata file at METAPATH. It is
+    /// OPTIONAL and can be omitted to reduce the snapshot metadata file size. In that case the
+    /// client MUST use a custom download limit for the listed metadata.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub length: Option<u64>,
+
+    /// TUF 4.4: HASHES is a dictionary that specifies one or more hashes of the metadata file at
+    /// METAPATH, including their cryptographic hash function. For example: { "sha256": HASH, ... }.
```

Comment:
```suggestion
    /// METAPATH, including their cryptographic hash function. For example: `{ "sha256": HASH, ... }`.
```
Should probably set off data example like this in code blocks, when inside docstrings.